RELEASES.md

```diff
@@ -1,3 +1,152 @@
+Version 1.51.0 (2021-03-25)
+============================
+
+Language
+--------
+- [You can now parameterize items such as functions, traits, and `struct`s by constant
```

Comment:
Perhaps it would make sense to explicitly use the term "const generics" here? Otherwise, I think some people will wonder if this is what is meant or not.